src/app.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const http = require('http');
+
+const requestParser = require('./lib/parse-request');
```

Comment:
You are requiring in parse-request and parse-body - but you have named the files parse_body and parse_request